kvs/src/kvs/server.cpp

```diff
@@ -523,33 +526,22 @@ void run(unsigned thread_id, Address public_ip, Address private_ip,
         kZmqUtil->send_string(serialized, &pushers[target_address]);
       }
 
-      // compute key access stats
+      // Get map of hot keys
```

Comment:
I wouldn't include comments that are clearly explained by the code itself. Comments should explain why things are the way they are. It's clear from the name `get_hot_map` that we are getting the hot map. If there's something unclear about why we might want to get the hot map, let's include that as a comment.